bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -238,6 +279,9 @@ spec:
             initialDelaySeconds: 20
             timeoutSeconds: 45
         {{- end }}
+        {{- if .Values.sidecars }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.sidecars "context" $) | nindent 8 }}
+        {{- end }}
       volumes:
         {{- if .Values.tlsEncryptionSecretName }}
         - name: encryption-secrets
```

Comment:
I cannot select that part of the code, but change this to use the tplValues.render macro

```
            name: {{ tpl .Values.existingConfiguration $ }}
```